addons/knobs/src/angular/helpers.js

```diff
@@ -1,22 +1,19 @@
 /* eslint no-underscore-dangle: 0 */
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { Component, SimpleChange, ChangeDetectorRef } from '@angular/core';
-import { getParameters, getAnnotations, getPropMetadata } from './utils';
+import { getParameters, getAnnotations } from './utils';
```

Comment:
May you now delete the `getPropMetadata ` from utils ?

Reply:
Done